### DISCUSSION TITLE
Any way to print sliced files from UVtools wirelessly on a Saturn 3 ultra?

### DISCUSSION THREAD

**Discussion by freaked1234** [sn4k3/UVtools#952] [Q&A]
Hey guys!
First of all I want to thank you for creating and maintaining this great tool!

Since my printer is pretty far away from the next computer I always use WiFi to start prints. As far as I understand there is no way to directly start prints from UVtools but I still hope that there is another way to remotely upload and start prints. 
Do you guys know a way to do this on a Saturn 3 Ultra?

**Comment by tennisdenis** [comment]
Have you found a solution yet? Would be interested. I have tried to reopen the edited goo-file with the tangoslicer to send it to the printer, but the slicer keeps crashing.

**Reply by sn4k3** [reply]
A solution: https://github.com/sn4k3/UVtools/issues/963
